rust/cyclotron-core/src/worker.rs

```diff
@@ -43,13 +45,15 @@ impl Worker {
         Ok(Self {
             pool,
             pending: Default::default(),
+            heartbeat_window: Duration::seconds(5),
```

Comment:
I suspect our p99.9 is below 1

Picked 5 for now as because janitor defaults to 30 for timeout, eventually this should all be configured or even communicated to workers by the janitor somehow